### PR TITLE
Event Page added

### LINKED ISSUE #18: Event Specific Page
**Is your feature request related to a problem? Please describe.**
No, its the initial website building part.

**Describe the solution you'd like**
Events specific page:
* Event title and poster. Then Register button design take it from already built design in accessories folder.
* Details and rules about the eventalong with date and time
* Then comes the organiser section and judges section.
image

**Describe alternatives you've considered**
Discuss the design alternatives with designers and project maintainers.

**Additional context**
Prototype: https://www.figma.com/proto/4a8ZglxR98YOEQpmXu93Ng/Dhruva-team-library?page-id=0%3A1&node-id=948%3A51&viewport=241%2C48%2C0.2&scaling=min-zoom

### PR DESCRIPTION
**Pull request by bindalsarthak03**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes #18 

## Type of change

Please delete options that are not relevant.

# Browsers Tested on?

- [x] Chrome
- [X] Firefox
- [X] Edge

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have checked the responsiveness of the component I built
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by faiz-hasan11** [comment]
- Link Navbar to your page
- In mobile screens cards should be aligned in a single column

**Comment by faiz-hasan11** [comment]
Development branch updated , take a pull and then work

**Review by faiz-hasan11 — DISMISSED** [review]
- In Mobile View Add some spaces between cards
- For iPad , make it a square grid of 2 rows and 2 columns

**Review by joshi008 — DISMISSED** [review]
- Allign event name, lorem ipsum, and register button more towards right. They are currently more left. 
image

- Event details is also too much left.

File: styles/Orgnjdge.module.css [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+@media (max-width: 601px) {
+    .carousel {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+    }
+    .card {
+        height: 90%;
+        width: 51%;
```
Comment (starwiz-7) [inline styles/Orgnjdge.module.css:176]: Remove width 51% here. It makes all cards show as a list.

File: styles/Orgnjdge.module.css [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    .taicon {
+        font-size: 40px;
+    }
+}
+
+@media (max-width: 601px) {
+    .carousel {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
```
Comment (starwiz-7) [inline styles/Orgnjdge.module.css:171]: change justify-content to space-evenly

File: components/Events/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import styles from '/styles/Orgnjdge.module.css'
+import { TiSocialLinkedinCircular, TiSocialTwitterCircular, TiSocialFacebookCircular } from 'react-icons/ti'
```
Comment (starwiz-7) [inline components/Events/Card.jsx]: Change the icons to FaLinkedin, FaTwitter, FaInstagram from react-icons/fa to maintain consistency as we have not used circular icons anywhere else.

File: styles/Orgnjdge.module.css [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    width: 98%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+
+.taicon {
+    font-size: 50px;
```
Comment (starwiz-7) [inline styles/Orgnjdge.module.css:35]: Change font-size to 30px and subsequently in media queries

Comment (starwiz-7) [reply styles/Orgnjdge.module.css:35]: You can refer to this [codesandbox](https://codesandbox.io/s/cool-tree-omcx4) to look at how the above changes will perform!

**Comment by joshi008** [comment]
image

**Comment by joshi008** [comment]
Also resolve the conflicts with the current devlopment branch

**Comment by faiz-hasan11** [comment]
Pull from the current dev branch and then push

**Review by starwiz-7 — APPROVED** [review]

**Pull request merged**